extensions/import/src/test/import.test.ts

```diff
@@ -123,17 +126,60 @@ describe('import extension tests', function (): void {
 			should(expectedConnectionValues).deepEqual(actualConnectionValues);
 
 		});
+
+		it('FileConfigPage - get schema returns active schema first', async () => {
```

Comment:
It doesn't usually matter for our case, but please try to use an actual function declaration instead of an arrow function (this goes for both describe/suite and it/test). See https://mochajs.org/#arrow-functions for why